### ISSUE TITLE
Non-parametric ANOVA

### ISSUE THREAD

**Issue by cjalbers** [jasp-stats/jasp-desktop#850]
Another implementation request. (All requests are so that next year I could use JASP rather than SPSS for my undergraduate course).

Add the option for a non-parametric t-test (Wilcoxon-Mann-Whitney test) and non-parametric one-way ANOVA (Kruskal.Wallis test; kruskal.test(dependentvariable ~ groupingvariable) ).

**Comment by SocProf** [comment]
I was going to make this request as well. Those of us who teach stats need to be able to perform non-parametric tests.

**Comment by vdsmagt** [comment]
May I add the Friedman to this enhancement request? Indeed If this functionality (both the Kruskal-Wallis and Friedman) is added I can finally get rid of SPSS for my undergrad stats course...

**Comment by EJWagenmakers** [comment]
Well that's a lofty goal, so we'll work hard to do this. @JohnnyDoorn, how does this look?

**Comment by JohnnyDoorn** [comment]
I just looked into this and it certainly looks doable for the next release, or the one after that. We would have to look into how the present the output though - we can do it similar to the t-test, where the nonparametric result is included in the table, or we can give the nonparametric ANOVA's a separate table.

**Comment by vdsmagt** [comment]
This of course is the developer's expertise but maybe a separate table might be easier since you can have a post-hoc analysis (DSCF, Dunn-bof and/or durbin conover, etc.) attached to that as well

**Comment by EJWagenmakers** [comment]
Yes, makes sense as a separate table, otherwise the output can quickly become confusing
E.J.

**Comment by alphonsasushma** [comment]
Are these requested options available in the current version of JASP. Just checking as I am using data which will need non-parametric analyses. Thank you.
